### ISSUE TITLE
Induction Furnace second output slot not served from Logistic pipe

### ISSUE THREAD

**Issue by ElVincenco** [xJon/Tekkit-2#65]
**Basic details**
- **Are you using a legitimate Minecraft account?**:yes
- **Are you using Java 8, 64-bit?**: Yes
- **Did you add any additional mods**: No
- **Crash/latest log link (if applicable)**:  not applicable

**Help needed**

Every IC2 device with 2 output slots has a problem. The second output slot get not cleared by a Logistic pipe. Is this a bug or do i need a IC2 upgrade for this to get cleared? 
And when requesting 200X Sand or other item that needs to be smelted the Logistic Pipes dont empty the Induction furnace until everything from the input chest is in the System. Is there a way to avoid issues like this?

**Comment by Lumpous** [comment]
Extracting from the secondary output works.
However, there seems to be a limit to how many actions LP can perform in a given time on a pipe, which can make it appear not to work for fast processes like the induction furnace.

This limit needs to be raised by using modules on the pipe.

I have tried to create a setup to show how this is done:
https://lumpous.de/tekkit2_65.mp4

This is an induction furnace designed to burn sand and cobble to glass and stone at the same time.
You will notice that it takes much longer to extract the stone and the glass is extracted regularly.
If I had placed more of the modules shown, the stone would be extracted more frequently.

Additionally it has to be mentioned that in my replica I placed an IC2C module in the furnace additionally.
